system_test/aest_sync_SUITE.erl

```diff
@@ -384,8 +390,35 @@ net_split_recovery(Cfg) ->
 
     ok.
 
+sync_speed(Cfg) ->
+    InitialNodes = [node_name(I) || I <- lists:seq(1, 4)],
+    NextNodes = [node_name(I) || I <- lists:seq(5, 6)],
+    Height = 1000,
+
+    NodeSpecs = [#{
+        name => N,
+        backend => aest_docker,
+        source => {pull, "aeternity/epoch:v0.11.1"},
+        mine_rate => 100
+    } || N <- InitialNodes ++ NextNodes],
```

Comment:
I see based on code inspection that absence of peers key in node spec means `peers: []` i.e. each node is alone.

Reply:
🤦‍♂️